src/mgr/BaseMgrModule.cc

```diff
@@ -386,30 +386,50 @@ ceph_option_get(BaseMgrModule *self, PyObject *args)
 }
 
 static PyObject*
-ceph_get_module_option(BaseMgrModule *self, PyObject *args)
+ceph_get_module_option_ex(BaseMgrModule *self, PyObject *args)
 {
+  char *module = nullptr;
   char *what = nullptr;
-  if (!PyArg_ParseTuple(args, "s:ceph_get_module_option", &what)) {
+  if (!PyArg_ParseTuple(args, "ss:ceph_get_module_option_ex", &module, &what)) {
     derr << "Invalid args!" << dendl;
     return nullptr;
   }
 
   PyThreadState *tstate = PyEval_SaveThread();
   std::string value;
-  bool found = self->py_modules->get_config(self->this_module->get_name(),
-      what, &value);
+  bool found = self->py_modules->get_config(module, what, &value);
 
   PyEval_RestoreThread(tstate);
 
   if (found) {
-    dout(10) << __func__ << " " << what << " found: " << value.c_str() << dendl;
+    dout(10) << __func__ << " " << module << "/" << what
+             << " found: " << value.c_str() << dendl;
     return PyString_FromString(value.c_str());
   } else {
-    dout(4) << __func__ << " " << what << " not found " << dendl;
+    dout(4) << __func__ << " " << module << "/" << what
+            << " not found " << dendl;
     Py_RETURN_NONE;
   }
 }
 
+static PyObject*
+ceph_get_module_option(BaseMgrModule *self, PyObject *args)
```

Comment:
My only suggestion here would be to add `const`-ness here to `self` and `args`, but as long as you're falling back to `ceph_get_module_ex`, you would need to do it in both places, and just had a look at C++ Python API and not many `const`s over there, so that'd be a hell.